docs/docs-beta/sidebars.ts

```diff
@@ -47,12 +47,12 @@ const sidebars: SidebarsConfig = {
           label: 'Ingestion and Transformation',
           link: {
             type: 'doc',
-            id: 'guides/etl',
+            id: 'guides/ingestion-and-transformation',
```

Comment:
I made this move to folder move to reflect the section title in the sidebar.

And also because I think "ETL" is kind of an ill-defined and confusing term. Its technical definition 
 – extract-transform-load – corresponds to a largely outdated pattern, where the data is transformed _before_ being loaded into the warehouse.